fix msisdn in s11 interface

pingping-lin opened the pull request.

krsna1729 commented on src/s11/handlers/create_session_handler.c:
```diff
@@ -156,7 +156,8 @@ create_session_processing()
 	ies[ieCount].length = htons(5);
 	ies[ieCount].instance = INSTANCE_ZERO;
-	memcpy(ies[ieCount].value, g_csReqInfo->MSISDN, 5);
+	//memcpy(ies[ieCount].value, g_csReqInfo->MSISDN, 5);
```
Why not delete this if its wrong?

pingping-lin replied:
solved in second patch

krsna1729 reviewed (commented):
Need commit message

ghost commented:
please retest

micpez commented:
retest this please

badhrinathpa commented:
+1

krsna1729 commented:
squash the commits

Commit:
fix msisdn in s11 interface

 According to 3gpp release 15,TS 29.274 V15.7.1, when transfer the
 MSISDN, there is a standarded packet format for each 4 bits or
 each digit. THis patch fixes the packet format issue.

fix msisdn in s11 interface
